manifests/vanilla/vsphere-csi-driver.yaml

```diff
@@ -102,6 +102,7 @@ data:
   "trigger-csi-fullsync": "false"
   "async-query-volume": "false"
   "csi-volume-manager-idempotency": "false"
+  "remove-vc-creds-on-topology-nodes": "false"
```

Comment:
Any other suggestions for the feature name ?

Reply:
Could not think of anything else. Should I name it `node-topology-without-vc-creds`?

Reply:
This sounds better: `node-topology-without-vc-creds`

Reply:
Cool, will change it.

Reply:
Done